Source/Core/Core/IOS/Network/WD/Command.cpp

```diff
@@ -8,29 +8,418 @@
 #include <cstring>
 #include <string>
 
+#include "Common/BitSet.h"
 #include "Common/CommonTypes.h"
 #include "Common/Logging/Log.h"
 #include "Common/Network.h"
 #include "Common/Swap.h"
-
 #include "Core/HW/Memmap.h"
 #include "Core/IOS/Network/MACUtils.h"
 
 namespace IOS::HLE::Device
 {
+namespace
+{
+// clang-format off
+//                           Channel:   FEDC BA98 7654 3210
```

Comment:
Where does this "Channel" value comes from? It reminds me an RFC IPv6 address: https://tools.ietf.org/html/rfc2732

Reply:
I believe this is just an easy reference for which bit is which channel in the two values below.

Reply:
I see, I guess that's not related to IPv6 address representation and prefix if that's the case:
https://tools.ietf.org/html/rfc1884
https://tools.ietf.org/html/rfc3513

(The newer versions of these drafts don't have this IPv6 placeholder)

Reply:
Yes, this is just to make it easier to see which channels are enabled.